docs(span): enable lint warnings on missing docs

overlookmotel opened the pull request:
This attribute was added in #6610, but then removed again in #6612. I assume removing it was an accident.

overlookmotel commented:
* **#6617** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6617?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @overlookmotel and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

DonIsaac commented:
### Merge activity

* **Oct 15, 7:55 PM EDT**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

overlookmotel committed:
docs(span): enable lint warnings on missing docs (#6617)

This attribute was added in #6610, but then removed again in #6612. I assume removing it was an accident.
